src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -833,8 +834,15 @@ class spell_pal_glyph_of_holy_light_dummy : public SpellScriptLoader
                 if (!healInfo || !healInfo->GetHeal())
                     return;
 
+                uint32 basePoints = healInfo->GetSpellInfo()->Effects[EFFECT_0].BasePoints + healInfo->GetSpellInfo()->Effects[EFFECT_0].DieSides;
```

Comment:
use SpellEffectInfo::CalcValue

Reply:
This is would return random value, while maximal is needed.